services/azure-servicebus/src/test/java/com/microsoft/windowsazure/services/servicebus/ServiceBusIntegrationTest.java

```diff
@@ -763,6 +765,29 @@ public void topicCanBeCreatedListedFetchedAndDeleted()
     }
 
     @Test
+    public void eventHubCanBeCreatedListedFetchedAndDeleted()
```

Comment:
Since the operations are quite dependent on each other (aka I would need to repeat most of the steps in every test), and they are quite slow (test takes about 16 sec ) I prefer to keep it in one. This also follows the same structure as other tests in this suite.